frontend/packages/console-shared/src/components/formik-fields/DropdownField.tsx

```diff
@@ -28,6 +29,7 @@ const DropdownField: React.FC<DropdownFieldProps> = ({ label, helpText, required
       <Dropdown
         {...props}
         id={fieldId}
+        autocompleteFilter={props.autoComplete ? fuzzy : undefined}
```

Comment:
Remove.

Reply:
DropdownField is used in other places like builder image Tag selection. So if it is not configurable we will end up having autocomplete in all the dropdowns.

eg: 
![image](https://user-images.githubusercontent.com/9964343/76480090-b7eb9200-6433-11ea-9d5b-a0844e9d67bd.png)

Reply:
updated